### PR TITLE
Add the registerCustomTemplate call

### PR DESCRIPTION
**Pull request by mcorbin**
```
./exo vm template register --checksum 6cf902921d1257a93cbcf8b23596692e --displaytext "Linux Debian 9 64-bit custom 30" --name "Linux Debian mcorbin 30 64-bit" --url "<url>" --username "debian" 
Registering the template...................................... success.
┼──────────┼────────────────────────────────┼──────────────────────────────────────┼
│   ZONE   │              NAME              │                  ID                  │
┼──────────┼────────────────────────────────┼──────────────────────────────────────┼
│ ch-gva-2 │ Linux Debian mcorbin 30 64-bit │ 4ba63720-cbbb-41d8-b3a2-da6543e5863a │
┼──────────┼────────────────────────────────┼──────────────────────────────────────┼
```

### CODE REVIEW

**Comment by falzm** [comment]
Any idea why "display text" is not called "description"? Is it not the same thing?

**Comment by mcorbin** [comment]
It is a description indeed, I will rename the flag (and hide cloudstack `displaytext`).

**Comment by mcorbin** [comment]
Call with description:

```
./exo vm template register --checksum 6cf902921d1257a93cbcf8b23596692e --description "Linux Debian 9 64-bit custom 30" --name "Linux Debian mcorbin 30 64-bit" --url "http://exoscale-packer-images.ppsos-ch-dk-2.exo.io/exoscale-debian-stretch-2019-03-27-6cf902.qcow2" --username "debian" 
Registering the template.................................................................................... success.
┼──────────┼────────────────────────────────┼──────────────────────────────────────┼
│   ZONE   │              NAME              │                  ID                  │
┼──────────┼────────────────────────────────┼──────────────────────────────────────┼
│ ch-gva-2 │ Linux Debian mcorbin 30 64-bit │ 9fae3c6c-e4b7-4a0e-a5ae-951bed5117fa │
┼──────────┼────────────────────────────────┼──────────────────────────────────────┼

```

**Comment by falzm** [comment]
> It is a description indeed, I will rename the flag (and hide cloudstack `displaytext`).

Ah it's a remnant from CloudStack... This explains everything 😏

File: cmd/template_register.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	return nil
+}
+
+func init() {
+	templateRegisterCmd.Flags().StringP("checksum", "", "", "the MD5 checksum value of this template")
+	templateRegisterCmd.Flags().StringP("description", "", "", "the template description")
+	templateRegisterCmd.Flags().StringP("name", "", "", "the name of the template")
+	templateRegisterCmd.Flags().StringP("zone", "", "", "the ID or name of the zone the template is to be hosted on")
+	templateRegisterCmd.Flags().StringP("url", "", "", "the URL of where the template is hosted")
+	templateRegisterCmd.Flags().BoolP("featured", "", false, "true if this template is a featured template, default is false")
```
Comment (falzm) [inline cmd/template_register.go]: Suggested simplification for the `--featured` flag description:

```suggestion
	templateRegisterCmd.Flags().BoolP("featured", "", false, "mark the template as featured")
```

File: cmd/template_register.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		table.Append([]string{template.ZoneName, template.Name, template.ID.String()})
+	}
+
+	table.Render()
+
+	return nil
+}
+
+func init() {
+	templateRegisterCmd.Flags().StringP("checksum", "", "", "the MD5 checksum value of this template")
```
Comment (falzm) [inline cmd/template_register.go]: Try to stay consistent in the flag descriptions, e.g. don't mix *this* and *the*:

```suggestion
	templateRegisterCmd.Flags().StringP("checksum", "", "", "the MD5 checksum value of the template")
```

Comment (mcorbin) [reply cmd/template_register.go]: Updated.

File: cmd/template_register.go (hunk 1 above)
Comment (mcorbin) [reply cmd/template_register.go]: Updated.

File: cmd/template_register.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+// templateRegisterCmd registers a template
+var templateRegisterCmd = &cobra.Command{
+	Use:     "register",
+	Short:   "register a custom template",
+	Aliases: gCreateAlias,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		checksum, err := cmd.Flags().GetString("checksum")
+		if err != nil {
+			fmt.Println("aa")
```
Comment (pierre-emmanuelJ) [inline cmd/template_register.go]: 👀

Comment (mcorbin) [reply cmd/template_register.go]: :O

Comment (falzm) [reply cmd/template_register.go]: Missed this one 😑

**Pull request merged**